src/applications/disability-benefits/2346/Components/ConfirmAddress.jsx

```diff
@@ -0,0 +1,101 @@
+import ErrorableTextInput from '@department-of-veterans-affairs/formation-react/ErrorableTextInput';
+import React, { useState } from 'react';
+import { Link } from 'react-router';
+
+export const ConfirmAddress = () => {
+  const [checked, setChecked] = useState(false);
+  const handleChange = e => {
+    const value = e.target.checked;
+    const name = e.target.name;
+
+    // eslint-disable-next-line no-debugger
+    debugger;
```

Comment:
We should remove this `debugger` 🙂

Reply:
ConfirmAddress.jsx is part of the 3614 branch work, so will update it there.